packages/website/src/templates/UpcomingEvent.js

```diff
@@ -0,0 +1,70 @@
+import { graphql } from 'gatsby'
+import * as React from 'react'
+import Layout from '../components/Layout/index'
+import MagazinePage from '../components/magazine'
+import Seo from '../components/SEO'
+
+const UpcomingEvent = ({ data }) => {
+  const nav = data?.prismicLayout?.data?.body
+  console.log(data?.prismicUpcomingEventPage?.data)
+  return (
+    <Layout navbar={nav}>
+      <Seo title="Technical Day" />
+      <MagazinePage data={data?.prismicUpcomingEventPage?.data} />
+    </Layout>
+  )
+}
+
+export default UpcomingEvent
```

Comment:
move this default export to the end.

Reply:
Is there any downside in keeping it just above the query?

Reply:
it looks a little less uniform to me.
Default exports are generally looked for at the end of the file.

Reply:
I think this is more of a preference. It is also a topic of discussion as it will involve changes in entire codebase.

Reply:
I agree.